### DISCUSSION TITLE
ZimaOs installed on ThinkPad 11e Laptop , Celeron N3450 4GBRAM 128GBSSD

### DISCUSSION THREAD

**Discussion by paffogorriones** [IceWhaleTech/CasaOS#1812] [Q&A]
Pretty excited , i have now ZimaOs installed in portable form , battery powerd and inbuild screen ehehe.

Does anyone has any advice on how i could control display on and off , or brightness, or how to keep the laptop on when i close the screen? I dont believe zima os has support for any of those , i just wondering if there is a way around it.

Addittionally terminal isn't giving me any output or feedback after I enter linux commands, do i need to anything particular to activate the terminal ? i mean i can see and type in to the terminal but nothing happen after that i press enter, it just go to a new line...

**Comment by nzGator** [comment]
I just installed mine on an old Dell Latitude and started looking for this as well. Not sure how to get to the command line yet but it is a matter of accessing power management via the powerprofilesctl.

**Comment by matbrff** [comment]
For anyone running ZimaOS on a laptop struggling with the device sleeping when the lid is closed, here is a working solution: 

1. login to SSH via Zima Client by
Settings > General > Developer mode > ssh web-based terminal (you have to configure it first, it's disabled by default)
2. enter the following command
`sudo sed -i 's/#HandleLidSwitch=suspend/HandleLidSwitch=ignore/' /etc/systemd/logind.conf && sudo systemctl restart systemd-logind`

Hope this helps someone! :)
